compile.py

```diff
@@ -189,98 +189,98 @@ def render_templates():
         ##########################
         # Carnival pages
         ##########################
-        "carnival/index.html": {
-            "title": "PennyLane Quantum Carnival",
-            "canonical_url": "https://pennylane.ai/carnival/index.html",
-            "description": "Join us for the PennyLane Quantum Carnival, "
-                           "running throughout the month of November 2021.",
-            "thumbnail": "https://pennylane.ai/img/carnival_logo_splash.png",
-            "faq": FAQ,
-            "root": ".."
-        },
-        "carnival/attractions.html": {
-            "title": "PennyLane Quantum Carnival &#8212; Attractions",
-            "canonical_url": "https://pennylane.ai/carnival/index.html",
-            "description": "Check out all the events taking place during "
-                           "the PennyLane Quantum Carnival.",
-            "thumbnail": "https://pennylane.ai/img/carnival_logo_splash.png",
-            "root": ".."
-        },
-        "carnival/talent_show.html": {
-            "title": "PennyLane Quantum Carnival &#8212; Talent Show",
-            "canonical_url": "https://pennylane.ai/carnival/index.html",
-            "description": "Step right up and show us what you've got. "
-                           "Submit your most creative work to our quantum Talent Show.",
-            "thumbnail": "https://pennylane.ai/img/carnival_logo_splash.png",
-            "root": ".."
-        },
-        "carnival/scavenger_hunt.html": {
-            "title": "PennyLane Quantum Carnival &#8212; Scavenger Hunt",
-            "canonical_url": "https://pennylane.ai/carnival/scavenger_hunt.html",
-            "description": "Take part in our festivities, collect tickets, "
-                           "and trade them in for prizes.",
-            "thumbnail": "https://pennylane.ai/img/carnival_logo_splash.png",
-            "root": ".."
-        },
-        "carnival/faq.html": {
-            "faq_page": True,
-            "title": "Frequently Asked Questions (FAQ) &#8212; PennyLane Quantum Carnival",
-            "canonical_url": "https://pennylane.ai/carnival/faq.html",
-            "description": "Curious? Find all the details about PennyLane Quantum Carnival.",
-            "thumbnail": "https://pennylane.ai/img/carnival_logo_splash.png",
-            "faq": CARNIVAL_FAQ,
-            "root": ".."
-        },
-        "carnival/register.html": {
-            "title": "Register &#8212; PennyLane Quantum Carnival",
-            "canonical_url": "https://pennylane.ai/carnival/register.html",
-            "description": "Sign up for the PennyLane Quantum Carnival.",
-            "thumbnail": "https://pennylane.ai/img/carnival_logo_splash.png",
-            "root": ".."
-        },
+#        "carnival/index.html": {
```

Comment:
@isaacdevlugt please revert this commenting before merging!

Reply:
All done & restored! 🚀